Framework/basic-aggregator.json

```diff
@@ -20,6 +20,10 @@
       },
       "conditionDB": {
         "url": "ccdb-test.cern.ch:8080"
+      },
+      "infologger": {                     "": "Configuration of the Infologger (optional).",
+        "filterDiscardDebug": "0",        "": "Set to 1 to discard debug and trace messages (default: 0)",
```

Comment:
Why 0/1, while in the rest of the config file we use true/false?

Reply:
It was because I took it directly from the infologger's parameters. You are right, it should be true/false.